### ISSUE TITLE
[BUG] gdf_csv_test.Dates failure in gpuCI

### ISSUE THREAD

**Issue by kkraus14** [rapidsai/cudf#676]
Getting failures in gpuCI with the following:

```
/home/jenkins/workspace/cudf-prb/cpp/tests/io/csv/csv_test.cu:456
Value of: ACol.hostdata()
Expected: has 10 elements where
element #0 is equal to 983750400000,
element #1 is equal to 1288483200000,
element #2 is equal to 782611200000,
element #3 is equal to 656208000000,
element #4 is equal to 0,
element #5 is equal to 798163200000,
element #6 is equal to 774144000000,
element #7 is equal to 1149638400000,
element #8 is equal to 1126828800000,
element #9 is equal to 2764800000
  Actual: {}
```

**Comment by j-ieong** [comment]
I haven't been able to repro locally yet.

**Comment by kkraus14** [comment]
Resolved in #677.
